plugins/pagerduty/komand_pagerduty/util/api.py

```diff
@@ -7,45 +7,117 @@
 
 class PagerDutyAPI:
     def __init__(self, api_key: str, logger: Logger):
-        self.headers = {"Authorization": f"Token token={api_key}", "Content-Type": "application/json"}
+        self.headers = {
+            "Authorization": f"Token token={api_key}",
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+        }
         self.session = requests.session()
         self.logger = logger
 
     def get_on_calls(self, schedule_id: str = None) -> dict:
```

Comment:
Can you include some docstrings for these functions? In the same format as we usually do:
```
<description>
:param schedule_id: <desc>
```
etc...

Reply:
docstrings have been added to this file